crates/frontend/src/pages/experiment_group_listing.rs

```diff
@@ -0,0 +1,637 @@
+                                    copied_ws.set(false);
+                                },
+                                Duration::new(1, 0),
+                            );
+                        }
+                        Err(_) => logging::log!("unable to copy to clipboard"),
+                    }
+                };
+                let description = get_row_string_fn(row, "description");
+
+                let change_reason = get_row_string_fn(row, "change_reason");
+                view! {
+                    <div>
+                        <A href=group_id.to_string() class="btn-link">
+                            {group_name}
+                        </A>
+                        <div class="text-gray-500">
+                            <span class="text-xs">
+                                {group_id}
+                            </span>
+                            <i class="ri-file-copy-line cursor-pointer ml-2" on:click:undelegated=handle_copy></i>
+                            <Show when=move || copied_rs.get()>
+                                <div class="inline-block bg-gray-600 ml-2 rounded-xl px-2">
+                                    <span class="text-white text-xs font-semibold">
+                                        "copied!"
+                                    </span>
+                                </div>
+                            </Show>
+                        </div>
+                    </div>
+                    <InfoDescription description=description change_reason=change_reason />
+                }.into_view()
+            },
+            ColumnSortable::Yes {
+                sort_fn: Callback::new(move |_| {
+                    let filters = filters_rws.get();
+                    let sort_by = filters.sort_by.unwrap_or_default().flip();
+                    let new_filters = ExpGroupFilters {
+                        sort_on: Some(SortOn::Name),
+                        sort_by: Some(sort_by),
+                        ..filters
+                    };
+                    filters_rws.set(new_filters);
+                }),
+                sort_by: current_sort_by.clone(),
+                currently_sorted: current_sort_on == SortOn::Name,
+            },
+            Expandable::Disabled,
+            default_column_formatter,
+        ),
+        Column::new(
+            "context".to_string(),
+            false,
+            move |_, row: &Map<String, Value>| {
+                let context = row
+                    .get("context")
+                    .and_then(|v| v.as_object().cloned())
+                    .unwrap_or_default();
+                let id = get_row_string_fn(row, "id");
+                let conditions =
+                    Conditions::from_context_json(&context).unwrap_or_default();
+
+                view! {
+                    <div class="w-[300px]">
+                        <ConditionComponent conditions grouped_view=false id />
+                    </div>
```

Comment:
ConditionComponent already accepts class, we dont need to wrap it in a div for this
```suggestion
                <ConditionComponent conditions grouped_view=false id class="w-[300px] />
```